flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/expressions/PlannerExpressionConverter.scala

```diff
@@ -37,16 +37,29 @@ import _root_.scala.collection.JavaConverters._
 class PlannerExpressionConverter private extends ApiExpressionVisitor[PlannerExpression] {
 
   override def visit(call: CallExpression): PlannerExpression = {
-    translateCall(call.getFunctionDefinition, call.getChildren.asScala)
+    val definition = call.getFunctionDefinition
+    translateCall(
+      definition, call.getChildren.asScala,
+      () => if (definition.getKind == FunctionKind.AGGREGATE ||
+        definition.getKind == FunctionKind.TABLE_AGGREGATE) {
```

Comment:
nit: fix indention

Reply:
Not sure if I get which part you don't like about the indentation. Did you mean:

```
    translateCall(
      definition, call.getChildren.asScala,
      () =>
        if (definition.getKind == FunctionKind.AGGREGATE ||
            definition.getKind == FunctionKind.TABLE_AGGREGATE) {
          ApiResolvedAggregateCallExpression(call)
        } else {
          ApiResolvedCallExpression(call)
        })
```